### ISSUE TITLE
Lyrics cache

### ISSUE THREAD

**Issue by Madh93** [emilioastarita/lyricfier#15]
I think the app should store lyrics in system as cache to avoid getting the same lyrics again.

**Comment by RobFog** [comment]
Does downloading lyrics suck that much bandwidth? Or is it about saving CPU time?

**Comment by Acen** [comment]
If there is an offline cache then this could be beneficial on low/no-bandwidth environments.

**Comment by emilioastarita** [comment]
Yes I was thinking in implement a plugin with more priority that look up for the song on disk and then the runner must save results on disk of the other plugins. But I didn't have time. If anyone wants to help with this idea I think that should work.
